### ISSUE TITLE
Bug when using -initlabel flag

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#2067]
~~~
sct_label_vertebrae -i sub-01_T1w.nii.gz -s sub-01_T1w_seg.nii.gz -c t2 -initlabel label_c2c3.nii.gz -qc ~/qc_spine_generic

--
Spinal Cord Toolbox (master/9d9b320757fc4a1db5bac3f706a01817c6b351c7)
Running /Users/julien/code/sct/scripts/sct_label_vertebrae.py -i sub-01_T1w.nii.gz -s sub-01_T1w_seg.nii.gz -c t2 -initlabel label_c2c3.nii.gz -qc /Users/julien/qc_spine_generic

Create temporary folder (/var/folders/6f/wy6ljmx9453cgth2qwv5l1s80000gn/T/sct-20181121162851.772298-label_vertebrae-cSnTHq)...

Copying input data to tmp folder...

Create label to identify disc...
WARNING: File /Users/julien/Desktop/ucl/sub-01/anat/labelz.nii.gz already exists. Will overwrite it.

DESCRIPTION
Perform mathematical operations on images. Some inputs can be either a number or a 4d image or 
several 3d images separated with ","

USAGE
sct_maths.pyc -i <file> -o <file_output>

MANDATORY ARGUMENTS
 -i <file>                    Input file.                   
 -o <file_output>             Output file.                  

OPTIONAL ARGUMENTS

Basic operations:
 -add <str>                   Add following input (can be number or image(s))
 -sub <str>                   Substract following input (can be number or image)
 -mul <str>                   Multiply following input (can be number or image(s))
 -div <str>                   Divide following input (can be number or image)
 -mean {x,y,z,t}              Average data across dimension.
 -rms {x,y,z,t}               Compute root-mean-squared across dimension.
 -std {x,y,z,t}               Compute STD across dimension. 
 -bin <float>                 Binarize image using specified threshold. E.g. -bin 0.5

Thresholding methods:
 -otsu <int>                  Threshold image using Otsu algorithm.
                                nbins: number of bins. Example: 256
 -otsu_adap <list of: int>    Threshold image using Adaptive Otsu algorithm.
                                block_size:
                                offset:
 -otsu_median <list of: int>  Threshold image using Median Otsu algorithm. Separate with ","
                                - Size of the median filter. Example: 2
                                - Number of iterations. Example: 3
                                
 -percent <int>               Threshold image using percentile of its histogram.
 -thr <float>                 Use following number to threshold image (zero below number).

Mathematical morphology
 -dilate <list of: int>       Dilate binary image. If only one input is given, structured element 
                              is a ball with input radius (in voxel). If comma-separated inputs are 
                              given (e.g., "2,4,5"), structured element is a box with input 
                              dimensions.
 -erode <list of: int>        Erode binary image. If only one input is given, structured element is 
                              a ball with input radius (in voxel). If comma-separated inputs are 
                              given (e.g., "2,4,5"), structured element is a box with input 
                              dimensions.

Filtering methods:
 -smooth <list of: float>     Gaussian smoothing filter with specified standard deviations in mm 
                              for each axis (e.g.: 2,2,1) or single value for all axis (e.g.: 2).
 -laplacian <list of: float>  Laplacian filtering with specified standard deviations in mm for all 
                              axes (e.g.: 2).
 -denoise <list of: str>      Non-local means adaptative denoising from P. Coupe et al. as 
                              implemented in dipy. Separate with ",". Example: p=1,b=3
                                p: (patch radius) similar patches in the non-local means are 
                                  searched for locally, inside a cube of side 2*p+1 centered at e
                                  ach voxel of interest. Default: p=1
                                b: (block radius) the size of the block to be used (2*b+1) in the 
                                  blockwise non-local means implementation. Default: b=5 (Block rad
                                  ius must be smaller than the smaller image dimension: default v
                                  alue is lowered for small images)
                                To use default parameters, write -denoise 1

Similarity metric
 -mi <file>                   Compute the mutual information (MI) between both input files (-i and 
                              -mi) as in: 
                              http://sciki
                              t-learn.org/
                              stable/modules/generated/sklearn.metrics.mutual_info_score.html
 -minorm <file>               Compute the normalized mutual information (MI) between both input 
                              files (-i and -mi) as in: 
                              http://scikit-learn.org/st
                              able/modules/generated/skl
                              earn.metrics.normalized_mutual_info_score.html
 -corr <file>                 Compute the cross correlation (CC) between both input files (-i and 
                              -cc).

Misc
 -symmetrize {0,1,2}          Symmetrize data along the specified dimension.
 -type {uint8,int16,int32,float32,complex64,float64,int8,uint16,uint32,int64,uint64}Output type.                  
 -v {0,1,2}                   Verbose. 0: nothing. 1: basic. 2: extended. Default value = 1

Command-line usage error: Option -i file doesn't exist: labelz.nii.gz
Aborted...
Total processing time: 0 min 8 s
~~~
